test/passes/stub-unsupported-js.txt

```diff
@@ -22,3 +22,23 @@
   )
  )
 )
+(module
+ (type $none_=>_none (func))
+ (type $none_=>_i32 (func (result i32)))
+ (type $none_=>_f32 (func (result f32)))
+ (table $0 1 1 funcref)
+ (elem (i32.const 0) $return-f32)
+ (func $return-f32 (result f32)
+  (f32.const 1)
+ )
+ (func $bad-indirect-call
+ )
```

Comment:
Why was the argument optimized out here? Shouldn't there be a drop? Same below, where it looks like the 0 argument was optimized out, then a new zero was inserted by `stubOut`?

Reply:
Ah, that 0 is not an argument, it's the function pointer index for the indirect call. So the first call has no params, and the second has a single i64.

Reply:
Ohhhhhh right. Thanks 👍

Reply:
Doesn't that index need to be in the generated block as well because its calculation could theoretically have side effects?

Reply:
Good point! Done.